grype/db/v6/distribution/client.go

```diff
@@ -230,15 +230,15 @@ func isSupersededBy(current *v6.Description, candidate v6.Description) bool {
 		return true
 	}
 
-	otherModelPart, otherOk := candidate.SchemaVersion.ModelPart()
-	currentModelPart, currentOk := current.SchemaVersion.ModelPart()
+	otherModelPart := candidate.SchemaVersion.Model
```

Comment:
nit: would this be clearer without temporary variables? e.g. `if candidate.SchemaVersion.Model == 0`?

Reply:
done -- also added a Valid() helper function instead of just a 0 check on the single model value